bitcoin/src/pow.rs

```diff
@@ -225,13 +225,13 @@ impl Target {
     /// [target]: crate::blockdata::block::Header::target
     #[cfg_attr(all(test, mutate), mutate)]
     pub fn difficulty(&self, network: Network) -> u128 {
+        debug_assert_ne!(self.0, U256::ZERO);
         let max = match network {
             Network::Bitcoin => Target::MAX_ATTAINABLE_MAINNET,
             Network::Testnet => Target::MAX_ATTAINABLE_TESTNET,
             Network::Signet => Target::MAX_ATTAINABLE_SIGNET,
             Network::Regtest => Target::MAX_ATTAINABLE_REGTEST,
         };
-        let d = max.0 / self.0;
```

Comment:
In c1ce8f9b296e59d467d16fa21704273f673424ca:

Did you mean to delete this line? The code does not compile without it.

Reply:
Bother, rebase fail. I'm getting smashed today.